src/lint/track_config.nim

```diff
@@ -228,8 +228,12 @@ const tags = [
 ].toHashSet()
 
 proc hasValidTags(data: JsonNode; path: Path): bool =
+  const errorAnnotation = "Tracks are annotated with tags to allow searching for tracks with certain tags.\n" &
+                          "The tags should be chosen based on the general usage of their language.\n" &
+                          "For more information and the full list of supported tags see:\n" &
+                          "https://exercism.org/docs/building/tracks/config-json#h-tags"
```

Comment:
This is the most important part, where we direct people to the documentation page.